conjur/cli.py

```diff
@@ -69,110 +77,125 @@ def run(self, *args):
+        formatter_class = lambda prog: argparse.RawTextHelpFormatter(prog,
+                                                                     max_help_position=60,
+                                                                     width=60)
+        # pylint: disable=line-too-long
+        parser = ArgparseWrapper(description=self.usage('conjur [global options] <command> <subcommand> [options] [args]'),
                                  epilog=self.main_epilog(),
                                  usage=argparse.SUPPRESS,
                                  add_help=False,
-                                 formatter_class=argparse.RawTextHelpFormatter)
+                                 formatter_class=formatter_class)
 
         global_optional = parser.add_argument_group("Global options")
         resource_subparsers = parser.add_subparsers(dest='resource', title=self.title("Commands"))
 
-        resource_subparsers.add_parser('whoami',
-            help='Provides information about the current logged-in user')
-
+        # pylint: disable=line-too-long
+        init_subparser = resource_subparsers.add_parser('init',
+                                                        help='Initialize the Conjur configuration',
+                                                        description=self.usage('conjur [global options] init [options] [args]'),
+                                                        epilog=self.command_epilog('conjur init -a my_org -u https://localhost\t'
+                                                                                   'Initialize conjur configuration and write to file (.conjurrc)'),
+                                                        usage=argparse.SUPPRESS,
+                                                        add_help=False,
+                                                        formatter_class=formatter_class)
+
+        init_options = init_subparser.add_argument_group(title=self.title("Options"))
+        init_options.add_argument('-a', '--account',
+                                  action='store', dest='account',
+                                  help='Provide Conjur organization account name ' \
+                                  '(obtained from server unless provided by this option)')
+        init_options.add_argument('-c', '--certificate',
+                                  action='store', dest='cert',
+                                  help='Provide Conjur SSL certificate file location')
+        init_options.add_argument('--force',
+                                  action='store_true',
+                                  dest='force', help='Force overwrite of existing files')
+        init_options.add_argument('-u', '--url',
+                                  action='store', dest='url',
+                                  help='Provide URL of Conjur server')
+        init_options.add_argument('-h', '--help', action='help', help='Display this help screen and exit')
```

Comment:
This code needs fine tuning.
You changed the code a bit.
It seems to me that the options used here are sometimes misused or redundant.
Please open a ticket to refactor this, and investigate the minimal correct usage necessary.